larray_editor/__init__.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import absolute_import, division, print_function
+
+try:
+    from larray_editor.api import *
+except ImportError:
+    def view(*args, **kwargs):
```

Comment:
wouldn't it be better to make larray_editor depend on pyqt/qtpy (in setup.py -- it is already the case in meta.yml) and avoid these dummy methods here? (we might want to keep something like this in larray though (at least temporarily).